capstone/src/test/java/com/google/univiz/EnumTest.java

```diff
@@ -0,0 +1,20 @@
+package com.google.univiz;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class EnumTest {
```

Comment:
It looks like this unit test is testing `CarnegieSizeDegree` class. So the test class name should be `CarnegieSizeDegreeTest`.